app/models/invenio_rdm_record_converter.rb

```diff
@@ -440,6 +445,19 @@ def normalize_date(date_string)
     end
   end
 
+  def rearrange_year(date_array)
+    if date_array[2].length == 2
+      # drawing the line at 22 as the year for when we no longer refer to 1900s
+      if date_array[2].to_i > 22
```

Comment:
I want to put the 22 into a constant or something similar to describe what it's doing, but I honestly, I can't really think of a good name. The comment is very helpful though.

Reply:
Haha, well comment or constant, we don't need this anymore. There are no dates with just "yy". They all include the century, "YYYY".